package.json

```diff
@@ -23,11 +23,11 @@
     "version": "grunt build && grunt copyForPublish && git add package.json release/* ",
     "postversion": "export GITTAG=\"echo $(git describe --abbrev=0 --tags | sed 's/^v//')\" && git push --force --set-upstream origin bumpVersion --follow-tags && git checkout gh-pages && git pull && cp out/connect-rtc-debug.js ./connect-rtc-debug-`$GITTAG`.js && cp out/connect-rtc.min.js ./connect-rtc-`$GITTAG`.min.js && ln -fs connect-rtc-debug-`$GITTAG`.js connect-rtc-debug-latest.js && ln -fs connect-rtc-`$GITTAG`.min.js connect-rtc-latest.min.js && git add connect-rtc*.js && git commit -m `$GITTAG` && git push --set-upstream origin gh-pages && git checkout master",
     "prepublish": "grunt build",
-    "test": "grunt && nyc --reporter=html mocha --compilers js:babel-core/register test/unit"
+    "test": "grunt && nyc --reporter=html mocha --compilers js:babel-core/register test/unit/**/*.js test/unit/*.js"
   },
   "_ucsdk_upgrade_note": "Please follow the instruction here(https://tiny.amazon.com/18mszf5px/quipaTMFStep) to upgrade @citrix/ucsdk",
   "dependencies": {
-    "@citrix/ucsdk": "2.0.5",
+    "@citrix/ucsdk": "3.1.0",
```

Comment:
Does this require legal team's approval?

Reply:
nope, I verified that the legal term did not change between 2.0.5 and 3.1.0